includes/CMB2_Utils.php

```diff
@@ -589,6 +589,7 @@ public static function concat_attrs( $attrs, $attr_exclude = array() ) {
 		foreach ( $attrs as $attr => $val ) {
 			$excluded = in_array( $attr, (array) $attr_exclude, true );
 			$empty    = false === $val && 'value' !== $attr;
+			$val 	  = is_array( $val ) ? implode( ',', $val ) : $val;
 			if ( ! $excluded && ! $empty ) {
 				// if data attribute, use single quote wraps, else double.
```

Comment:
Move the value handling to inside the condition.
```suggestion
			if ( ! $excluded && ! $empty ) {
				$val = is_array( $val ) ? implode( ',', $val ) : $val;

				// if data attribute, use single quote wraps, else double.
```